### ISSUE TITLE
UndefVarError: `make_block_docstring` not defined

### ISSUE THREAD

**Issue by gdalle** [JuliaPlots/MakieTeX.jl#54]
I tried to use MakieTeX today and I can't even load the package

```julia
(@v1.10) pkg> activate --temp
  Activating new project at `/tmp/jl_mU4N3j`

(jl_mU4N3j) pkg> add MakieTeX
   Resolving package versions...
    Updating `/tmp/jl_mU4N3j/Project.toml`
  [6d554a22] + MakieTeX v0.4.1
    Updating `/tmp/jl_mU4N3j/Manifest.toml`
  [621f4979] + AbstractFFTs v1.5.0
  [1520ce14] + AbstractTrees v0.4.5
  [79e6a3ab] + Adapt v4.0.4
  [35492f91] + AdaptivePredicates v1.1.1
  [66dad0bd] + AliasTables v1.1.3
  [27a7e980] + Animations v0.4.1
  [67c07d97] + Automa v1.0.4
  [13072b0f] + AxisAlgorithms v1.1.0
  [39de3d68] + AxisArrays v0.4.7
  [fa961155] + CEnum v0.5.0
  [159f3aea] + Cairo v1.0.5
  [49dc2e85] + Calculus v0.5.1
  [d360d2e6] + ChainRulesCore v1.24.0
  [a2cac450] + ColorBrewer v0.4.0
  [35d6a980] + ColorSchemes v3.26.0
  [3da002f7] + ColorTypes v0.11.5
  [c3611d14] + ColorVectorSpace v0.10.0
  [5ae59095] + Colors v0.12.11
  [34da2185] + Compat v4.16.0
  [187b0558] + ConstructionBase v1.5.6
  [d38c429a] + Contour v0.6.3
  [9a962f9c] + DataAPI v1.16.0
  [864edb3b] + DataStructures v0.18.20
  [e2d170a0] + DataValueInterfaces v1.0.0
  [927a84f5] + DelaunayTriangulation v1.1.3
  [31c24e10] + Distributions v0.25.110
  [ffbed154] + DocStringExtensions v0.9.3
  [fa6b7ba4] + DualNumbers v0.6.8
  [4e289a0a] + EnumX v1.0.4
  [429591f6] + ExactPredicates v2.2.8
  [411431e0] + Extents v0.1.3
  [7a1cc6ca] + FFTW v1.8.0
  [5789e2e9] + FileIO v1.16.3
  [8fc22ac5] + FilePaths v0.8.3
  [48062228] + FilePathsBase v0.9.21
  [1a297f60] + FillArrays v1.11.0
  [53c48c17] + FixedPointNumbers v0.8.5
  [1fa38f19] + Format v1.3.7
  [b38be410] + FreeType v4.1.1
  [663a7486] + FreeTypeAbstraction v0.10.3
  [cf35fbd7] + GeoInterface v1.3.5
  [5c1252a2] + GeometryBasics v0.4.11
  [a2bd30eb] + Graphics v1.1.2
  [3955a311] + GridLayoutBase v0.11.0
  [42e2da0e] + Grisu v1.0.2
  [34004b35] + HypergeometricFunctions v0.3.23
  [2803e5a7] + ImageAxes v0.6.11
  [c817782e] + ImageBase v0.1.7
  [a09fc81d] + ImageCore v0.10.2
  [82e4d734] + ImageIO v0.6.8
  [bc367c6b] + ImageMetadata v0.9.9
  [9b13fd28] + IndirectArrays v1.0.0
  [d25df0c9] + Inflate v0.1.5
  [a98d9a8b] + Interpolations v0.15.1
  [d1acc4aa] + IntervalArithmetic v0.22.14
  [8197267c] + IntervalSets v0.7.10
  [92d709cd] + IrrationalConstants v0.2.2
  [f1662d9f] + Isoband v0.1.1
  [c8e1da08] + IterTools v1.10.0
  [82899510] + IteratorInterfaceExtensions v1.0.0
  [692b3bcd] + JLLWrappers v1.5.0
  [682c06a0] + JSON v0.21.4
  [b835a17e] + JpegTurbo v0.1.5
  [5ab0869b] + KernelDensity v0.6.9
  [b964fa9f] + LaTeXStrings v1.3.1
  [8cdb02fc] + LazyModules v0.3.1
  [2ab3a3ac] + LogExpFunctions v0.3.28
  [1914dd2f] + MacroTools v0.5.13
  [ee78f7c6] + Makie v0.21.5
  [20f20a25] + MakieCore v0.8.4
  [6d554a22] + MakieTeX v0.4.1
  [dbb5928d] + MappedArrays v0.4.2
  [0a4f8689] + MathTeXEngine v0.6.1
  [e1d29d7a] + Missings v1.2.0
  [e94cdb99] + MosaicViews v0.3.4
  [77ba4419] + NaNMath v1.0.2
  [f09324ee] + Netpbm v1.1.1
  [510215fc] + Observables v0.5.5
  [6fe1bfb0] + OffsetArrays v1.14.1
  [52e1d378] + OpenEXR v0.3.2
  [bac558e1] + OrderedCollections v1.6.3
  [90014a1f] + PDMats v0.11.31
  [f57f5aa1] + PNGFiles v0.4.3
  [19eb6ba3] + Packing v0.5.0
  [5432bcbf] + PaddedViews v0.5.12
  [69de0a69] + Parsers v2.8.1
  [eebad327] + PkgVersion v0.3.3
  [995b91a9] + PlotUtils v1.4.1
  [647866c9] + PolygonOps v0.1.2
  [aea7be01] + PrecompileTools v1.2.1
  [21216c6a] + Preferences v1.4.3
  [92933f4c] + ProgressMeter v1.10.2
  [43287f4e] + PtrArrays v1.2.0
  [4b34888f] + QOI v1.0.0
  [1fd47b50] + QuadGK v2.10.1
  [b3c3ace0] + RangeArrays v0.3.2
  [c84ed2f1] + Ratios v0.4.5
  [189a3867] + Reexport v1.2.2
  [05181044] + RelocatableFolders v1.0.1
  [ae029012] + Requires v1.3.0
  [79098fc4] + Rmath v0.7.1
  [5eaf0fd0] + RoundingEmulator v0.2.1
  [c4c386cf] + Rsvg v1.0.0
  [fdea26ae] + SIMD v3.5.0
  [6c6a2e73] + Scratch v1.2.1
  [65257c39] + ShaderAbstractions v0.4.1
  [992d4aef] + Showoff v1.0.3
  [73760f76] + SignedDistanceFields v0.4.0
  [699a6c99] + SimpleTraits v0.9.4
  [45858cf5] + Sixel v0.1.3
  [a2af1166] + SortingAlgorithms v1.2.1
  [276daf66] + SpecialFunctions v2.4.0
  [cae243ae] + StackViews v0.1.1
  [90137ffa] + StaticArrays v1.9.7
  [1e83bf80] + StaticArraysCore v1.4.3
  [82ae8749] + StatsAPI v1.7.0
  [2913bbd2] + StatsBase v0.34.3
  [4c63d2b9] + StatsFuns v1.3.1
  [09ab397b] + StructArrays v0.6.18
  [3783bdb8] + TableTraits v1.0.1
  [bd369af6] + Tables v1.12.0
  [62fd8b95] + TensorCore v0.1.1
  [731e570b] + TiffImages v0.10.0
  [3bb67fe8] + TranscodingStreams v0.11.2
  [981d1d27] + TriplotBase v0.1.0
  [f0ed7684] + Typstry v0.2.0
  [1cfade01] + UnicodeFun v0.4.1
  [1986cc42] + Unitful v1.21.0
  [efce3f68] + WoodburyMatrices v1.0.0
  [6e34b625] + Bzip2_jll v1.0.8+1
  [4e9b3aee] + CRlibm_jll v1.0.1+0
  [83423d85] + Cairo_jll v1.18.0+2
  [5ae413db] + EarCut_jll v2.2.4+0
  [2e619515] + Expat_jll v2.6.2+0
  [b22a6f82] + FFMPEG_jll v6.1.1+0
  [f5851436] + FFTW_jll v3.3.10+0
  [a3f928ae] + Fontconfig_jll v2.13.96+0
  [d7e528f0] + FreeType2_jll v2.13.2+0
  [559328eb] + FriBidi_jll v1.0.14+0
  [78b55507] + Gettext_jll v0.21.0+0
  [61579ee1] + Ghostscript_jll v9.55.0+4
  [7746bdde] + Glib_jll v2.80.2+0
  [3b182d85] + Graphite2_jll v1.3.14+0
  [655565e8] + HarfBuzz_ICU_jll v2.8.1+0
  [2e76f6c2] + HarfBuzz_jll v2.8.1+1
⌅ [a51ab1cf] + ICU_jll v69.1.0+0
  [905a6f67] + Imath_jll v3.1.11+0
  [1d5cc7b8] + IntelOpenMP_jll v2024.2.0+0
  [aacddb02] + JpegTurbo_jll v3.0.3+0
  [c1c5ebd0] + LAME_jll v3.100.2+0
⌅ [88015f11] + LERC_jll v3.0.0+1
  [1d63c593] + LLVMOpenMP_jll v15.0.7+0
  [dd4b983a] + LZO_jll v2.10.2+0
⌅ [e9f186c6] + Libffi_jll v3.2.2+1
  [d4300ac3] + Libgcrypt_jll v1.8.11+0
  [7add5ba3] + Libgpg_error_jll v1.49.0+0
  [94ce4f54] + Libiconv_jll v1.17.0+0
  [4b2f31a3] + Libmount_jll v2.40.1+0
  [925c91fb] + Librsvg_jll v2.54.5+0
  [89763e89] + Libtiff_jll v4.6.0+0
  [38a345b3] + Libuuid_jll v2.40.1+0
  [d3a379c0] + LittleCMS_jll v2.16.0+0
  [856f044c] + MKL_jll v2024.2.0+0
  [e7412a2a] + Ogg_jll v1.3.5+1
  [18a262bb] + OpenEXR_jll v3.2.4+0
  [643b3616] + OpenJpeg_jll v2.5.2+0
  [458c3c95] + OpenSSL_jll v3.0.14+0
  [efe28fd5] + OpenSpecFun_jll v0.5.5+0
  [91d4177d] + Opus_jll v1.3.2+0
  [36c8627f] + Pango_jll v1.52.2+0
  [30392449] + Pixman_jll v0.43.4+0
⌅ [9c32591e] + Poppler_jll v23.12.0+1
  [f50d1b31] + Rmath_jll v0.4.3+0
  [eb4b1da6] + Typst_jll v0.11.1+0
  [02c8fc9c] + XML2_jll v2.13.1+0
  [aed1982a] + XSLT_jll v1.1.41+0
  [ffd25f8a] + XZ_jll v5.4.6+0
  [4f6342f7] + Xorg_libX11_jll v1.8.6+0
  [0c0b7dd1] + Xorg_libXau_jll v1.0.11+0
  [a3789734] + Xorg_libXdmcp_jll v1.1.4+0
  [1082639a] + Xorg_libXext_jll v1.3.6+0
  [ea2f1a96] + Xorg_libXrender_jll v0.9.11+0
  [14d82f49] + Xorg_libpthread_stubs_jll v0.1.1+0
  [c7cfdc94] + Xorg_libxcb_jll v1.17.0+0
  [c5fb5394] + Xorg_xtrans_jll v1.5.0+0
  [3161d3a3] + Zstd_jll v1.5.6+0
  [da03df04] + gdk_pixbuf_jll v2.42.10+0
  [9a68df92] + isoband_jll v0.2.3+0
  [a4ae2306] + libaom_jll v3.9.0+0
  [0ac62f75] + libass_jll v0.15.1+0
  [f638f0a6] + libfdk_aac_jll v2.0.2+0
  [b53b4c65] + libpng_jll v1.6.43+1
  [075b6546] + libsixel_jll v1.10.3+0
  [f27f6e37] + libvorbis_jll v1.3.7+2
  [1317d2d5] + oneTBB_jll v2021.12.0+0
  [d7dd28d6] + tectonic_jll v0.14.1+0
  [1270edf5] + x264_jll v2021.5.5+0
  [dfaa095f] + x265_jll v3.5.0+0
  [0dad84c5] + ArgTools v1.1.1
  [56f22d72] + Artifacts
  [2a0f44e3] + Base64
  [8bf52ea8] + CRC32c
  [ade2ca70] + Dates
  [8ba89e20] + Distributed
  [f43a241f] + Downloads v1.6.0
  [7b1f6079] + FileWatching
  [b77e0a4c] + InteractiveUtils
  [4af54fe1] + LazyArtifacts
  [b27032c2] + LibCURL v0.6.4
  [76f85450] + LibGit2
  [8f399da3] + Libdl
  [37e2e46d] + LinearAlgebra
  [56ddb016] + Logging
  [d6f4376e] + Markdown
  [a63ad114] + Mmap
  [ca575930] + NetworkOptions v1.2.0
  [44cfe95a] + Pkg v1.10.0
  [de0858da] + Printf
  [3fa0cd96] + REPL
  [9a3f8284] + Random
  [ea8e919c] + SHA v0.7.0
  [9e88b42a] + Serialization
  [1a1011a3] + SharedArrays
  [6462fe0b] + Sockets
  [2f01184e] + SparseArrays v1.10.0
  [10745b16] + Statistics v1.10.0
  [4607b0f0] + SuiteSparse
  [fa267f1f] + TOML v1.0.3
  [a4e569a6] + Tar v1.10.0
  [8dfed614] + Test
  [cf7118a7] + UUIDs
  [4ec0a83e] + Unicode
  [e66e0078] + CompilerSupportLibraries_jll v1.1.1+0
  [deac9b47] + LibCURL_jll v8.4.0+0
  [e37daf67] + LibGit2_jll v1.6.4+0
  [29816b5a] + LibSSH2_jll v1.11.0+1
  [c8ffd9c3] + MbedTLS_jll v2.28.2+1
  [14a3606d] + MozillaCACerts_jll v2023.1.10
  [4536629a] + OpenBLAS_jll v0.3.23+4
  [05823500] + OpenLibm_jll v0.8.1+2
  [efcefdf7] + PCRE2_jll v10.42.0+1
  [bea87d4a] + SuiteSparse_jll v7.2.1+1
  [83775a58] + Zlib_jll v1.2.13+1
  [8e850b90] + libblastrampoline_jll v5.8.0+1
  [8e850ede] + nghttp2_jll v1.52.0+1
  [3f19e933] + p7zip_jll v17.4.0+2
        Info Packages marked with ⌅ have new versions available but compatibility constraints restrict them from upgrading. To see why use `status --outdated -m`
Precompiling project...
  ✗ MakieTeX
  0 dependencies successfully precompiled in 10 seconds. 236 already precompiled.
  1 dependency errored.
  For a report of the errors see `julia> err`. To retry use `pkg> precompile`

julia> using MakieTeX
Precompiling MakieTeX
        Info Given MakieTeX was explicitly requested, output will be shown live 
ERROR: LoadError: UndefVarError: `make_block_docstring` not defined
Stacktrace:
 [1] top-level scope
   @ ~/.julia/packages/Makie/rEu75/src/makielayout/blocks.jl:114
 [2] include(mod::Module, _path::String)
   @ Base ./Base.jl:495
 [3] include(x::String)
   @ MakieTeX ~/.julia/packages/MakieTeX/KAivD/src/MakieTeX.jl:1
 [4] top-level scope
   @ ~/.julia/packages/MakieTeX/KAivD/src/MakieTeX.jl:35
 [5] include
   @ ./Base.jl:495 [inlined]
 [6] include_package_for_output(pkg::Base.PkgId, input::String, depot_path::Vector{String}, dl_load_path::Vector{String}, load_path::Vector{String}, concrete_deps::Vector{Pair{Base.PkgId, UInt128}}, source::Nothing)
   @ Base ./loading.jl:2222
 [7] top-level scope
   @ stdin:3
in expression starting at /home/gdalle/.julia/packages/MakieTeX/KAivD/src/layoutable.jl:5
in expression starting at /home/gdalle/.julia/packages/MakieTeX/KAivD/src/MakieTeX.jl:1
in expression starting at stdin:3
  ✗ MakieTeX
  0 dependencies successfully precompiled in 6 seconds. 236 already precompiled.

ERROR: The following 1 direct dependency failed to precompile:

MakieTeX [6d554a22-29e7-47bd-aee5-0c5f06619414]

Failed to precompile MakieTeX [6d554a22-29e7-47bd-aee5-0c5f06619414] to "/home/gdalle/.julia/compiled/v1.10/MakieTeX/jl_YR9h8O".
ERROR: LoadError: UndefVarError: `make_block_docstring` not defined
Stacktrace:
 [1] top-level scope
   @ ~/.julia/packages/Makie/rEu75/src/makielayout/blocks.jl:114
 [2] include(mod::Module, _path::String)
   @ Base ./Base.jl:495
 [3] include(x::String)
   @ MakieTeX ~/.julia/packages/MakieTeX/KAivD/src/MakieTeX.jl:1
 [4] top-level scope
   @ ~/.julia/packages/MakieTeX/KAivD/src/MakieTeX.jl:35
 [5] include
   @ ./Base.jl:495 [inlined]
 [6] include_package_for_output(pkg::Base.PkgId, input::String, depot_path::Vector{String}, dl_load_path::Vector{String}, load_path::Vector{String}, concrete_deps::Vector{Pair{Base.PkgId, UInt128}}, source::Nothing)
   @ Base ./loading.jl:2222
 [7] top-level scope
   @ stdin:3
in expression starting at /home/gdalle/.julia/packages/MakieTeX/KAivD/src/layoutable.jl:5
in expression starting at /home/gdalle/.julia/packages/MakieTeX/KAivD/src/MakieTeX.jl:1
in expression starting at stdin:
Stacktrace:
  [1] pkgerror(msg::String)
    @ Pkg.Types ~/.julia/juliaup/julia-1.10.4+0.x64.linux.gnu/share/julia/stdlib/v1.10/Pkg/src/Types.jl:70
  [2] precompile(ctx::Pkg.Types.Context, pkgs::Vector{…}; internal_call::Bool, strict::Bool, warn_loaded::Bool, already_instantiated::Bool, timing::Bool, _from_loading::Bool, kwargs::@Kwargs{…})
    @ Pkg.API ~/.julia/juliaup/julia-1.10.4+0.x64.linux.gnu/share/julia/stdlib/v1.10/Pkg/src/API.jl:1659
  [3] precompile(pkgs::Vector{Pkg.Types.PackageSpec}; io::Base.TTY, kwargs::@Kwargs{_from_loading::Bool})
    @ Pkg.API ~/.julia/juliaup/julia-1.10.4+0.x64.linux.gnu/share/julia/stdlib/v1.10/Pkg/src/API.jl:159
  [4] precompile
    @ ~/.julia/juliaup/julia-1.10.4+0.x64.linux.gnu/share/julia/stdlib/v1.10/Pkg/src/API.jl:147 [inlined]
  [5] #precompile#114
    @ ~/.julia/juliaup/julia-1.10.4+0.x64.linux.gnu/share/julia/stdlib/v1.10/Pkg/src/API.jl:146 [inlined]
  [6] #invokelatest#2
    @ ./essentials.jl:894 [inlined]
  [7] invokelatest
    @ ./essentials.jl:889 [inlined]
  [8] _require(pkg::Base.PkgId, env::String)
    @ Base ./loading.jl:1963
  [9] __require_prelocked(uuidkey::Base.PkgId, env::String)
    @ Base ./loading.jl:1812
 [10] #invoke_in_world#3
    @ ./essentials.jl:926 [inlined]
 [11] invoke_in_world
    @ ./essentials.jl:923 [inlined]
 [12] _require_prelocked(uuidkey::Base.PkgId, env::String)
    @ Base ./loading.jl:1803
 [13] macro expansion
    @ ./loading.jl:1790 [inlined]
 [14] macro expansion
    @ ./lock.jl:267 [inlined]
 [15] __require(into::Module, mod::Symbol)
    @ Base ./loading.jl:1753
 [16] #invoke_in_world#3
    @ ./essentials.jl:926 [inlined]
 [17] invoke_in_world
    @ ./essentials.jl:923 [inlined]
 [18] require(into::Module, mod::Symbol)
    @ Base ./loading.jl:1746
Some type information was truncated. Use `show(err)` to see complete types.
```

**Comment by asinghvi17** [comment]
Does current master fix the issue?  This is a known bug in Makie and there's a PR up to fix it there, I've monkeypatched it for now.

**Comment by gdalle** [comment]
It works, thanks!

**Comment by asinghvi17** [comment]
Fixed by 632c0d7
